admin/app/dfp/ApiHelper.scala

```diff
@@ -16,26 +16,32 @@ private[dfp] object ApiHelper extends GuLogging {
         }
       }
     }
-
     (dfpLineItem.getRoadblockingType == RoadblockingType.CREATIVE_SET) &&
     hasA1x1Pixel(dfpLineItem.getCreativePlaceholders)
   }
 
-  def toJodaTime(time: DateTime): JodaDateTime = {
-    val date = time.getDate
-    new JodaDateTime(
-      date.getYear,
-      date.getMonth,
-      date.getDay,
-      time.getHour,
-      time.getMinute,
-      time.getSecond,
-      DateTimeZone.forID(time.getTimeZoneId),
-    )
-  }
-
   def toSeq[A](as: Array[A]): Seq[A] = Option(as) map (_.toSeq) getOrElse Nil
 
   //noinspection IfElseToOption
   def optJavaInt(i: java.lang.Integer): Option[Int] = if (i == null) None else Some(i)
+
+  def toLocalDateTime(gdt: com.google.api.ads.admanager.axis.v202011.DateTime): LocalDateTime = {
```

Comment:
I'd find an alias import of this kind of `DateTime` a bit easier to read probably. `DfpDateTime` for example.

Reply:
Oh yeah, thanks for pointing that out. It took me a moment to realise that Google had their own DateTime and I wanted to make it clear. Let me refactor a bit.

Reply:
Done